ONSdigital/eq-author
# 341 Delete page

### What is the context of this PR?

[Trello card](https://trello.com/c/E2iEO5GW/341-delete-page-5)

- adds toolbar to editor
- clicking delete button opens modal confirmation

### How to review 

- navigate to or add a question
- click delete
- confirm modal opens
- confirm "delete" button deletes that question and navigates as expected
- confirm action is cancelable
- confirm navigation via back/prev buttons closes modal
